modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -95,6 +101,17 @@ object types {
           ),
           remainingWeight = Weight(30L) // facilitators
         )
+      case _ =>
+        ProgramsDistributionConfig(
+          weights = Map(
+            stardustNewPrimary -> Weight(5L),
+            stardustSecondary -> Weight(5L),
+            testnet -> Weight(3L),
+            dataPool -> Weight(55L),
+            integrationNet -> Weight(15L)
+          ),
+          remainingWeight = Weight(17L) // facilitators
+        )
```

Comment:
@ryle-goehausen Are `stardustNewPrimary` and `integrationNet` the two new coins that we wanted to reward with this PR?

If they are being added to this new case statement does that mean the `OneTimeReward` thing I added is not necessary since this is how they're being rewarded?

Reply:
Will get clarification from Alex on first point, I assume this is correct since it matches prior behavior and 303 doesn't provide enough detail to answer yet.

One Time Reward is still required no matter what this config is, as the existing balance must be transferred from the current unspendable address.

Reply:
Just got clarification, this first part is correct, 5% to new primary, 5% to secondary.